Tienda-Virtual/src/app/modules/product/component/category/category.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component } from '@angular/core';
+import { CategoryService } from '../../_service/category.service';
+import { Category } from '../../_model/category';
+
+@Component({
+  selector: 'app-category',
+  standalone: true,
+  imports: [],
+  templateUrl: './category.component.html',
+  styleUrl: './category.component.css'
+})
+export class CategoryComponent {
+
+  categories: Category[]= [];
+  constructor(private categoryService:CategoryService){}
+
+  ngOnInit(){
+    this.categories = this.categoryService.getCategories();
```

Comment:
Basta con llamar a la función que ya hace lo mismo:
```suggestion
    this.getCategories();
```